engine/src/main/java/org/terasology/rendering/nui/animation/AbstractAnimation.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2014 MovingBlocks
```

Comment:
Nope, just change the year to 2016. If you've created this file from IntelliJ, a license header with the current year would be autogenerated,